test1.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+import fnmatch
+import os
+import random
+import string 
+
+file1 = 'GoogleMap.java'
+file2 = 'aaaaa.java'
+file3 = 'rcrcrr.java'
+file4 = 'd.java'
+file5 = 'FB.java'
+file6 = 'cf.java'
+file7 = 'ghg%jjh.java'
+file8 = 'R.java'
+class TestProject(unittest.TestCase):
+	def setUp(self):
+		pass
+		
+	def test_filename_oneChar_GoogleMap(self):
+		self.assertFalse(fnmatch.fnmatch(file1,'[a-z].java'))

```

Comment:
fnmatch.fnmatch(file, '[a-z].java') can be isolated in a helper methode called isFileNameLowCharacters
for all the concerned test cases